src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -700,5 +706,13 @@ public static void RaiseTestRunWarning(string warningMessage)
 
             Output.Warning(AppendPrefix, warningMessage);
         }
+
+        /// <summary>
+        /// updates the test results
+        /// </summary>
+        public void UpdateTestsStatus()
+        {
+            testsStatus = "| Executed :" + testsTotal + "  | Passed :" + testsPassed + "  | Failed: " + testsFailed + "  | Skipped: " + testsSkipped + "";
```

Comment:
`Executed_:` and `Passed_:` should have no space before the colon and one space after: `Executed:_` and `Passed:_` (replace insert space for "_").